play-services-base/core/src/main/kotlin/org/microg/gms/settings/SettingsContract.kt

```diff
@@ -34,6 +34,7 @@ object SettingsContract {
         const val SECURITY_TOKEN = "securityToken"
         const val VERSION_INFO = "versionInfo"
         const val DEVICE_DATA_VERSION_INFO = "deviceDataVersionInfo"
+        const val HIDE_APP_ICON = "hideAppIcon"
```

Comment:
This does not belong into the Checkin settings contract. In fact it's not needed at all in `SettingsContract` or `SettingsProvider`, see my other comment